### PR TITLE
media-video/webcamoid: add 9.0.0 

### PR DESCRIPTION
**Pull request by BlackCrypto103**

### CODE REVIEW

File: media-video/webcamoid/webcamoid-9.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	>=sys-kernel/linux-headers-3.6
+"
+
+src_configure() {
+	#Disable git in package source. If not disabled the cmake configure process will show a lot of "fatal not a git repository" errors
+	sed 's|find_program(GIT_BIN git)|#find_program(GIT_BIN git)|' libAvKys/cmake/ProjectCommons.cmake -i
+
+	local mycmakeargs=(
+		"-S ${WORKDIR}/${PF}"
+		"-B ${WORKDIR}/${PF}/webcamoid-build"
```
Comment (juippis) [inline media-video/webcamoid/webcamoid-9.0.0.ebuild:50]: I'm pretty sure setting these are dangerous. Is there a reason you added them?

File: media-video/webcamoid/webcamoid-9.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+DESCRIPTION="A full featured webcam capture application"
+HOMEPAGE="https://webcamoid.github.io"
+SRC_URI="https://github.com/webcamoid/${PN}/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+RESTRICT="mirror"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+CMAKE_USE_DIR="${WORKDIR}/${PF}"
+BUILD_DIR="${CMAKE_USE_DIR}/${PN}-build"
```
Comment (juippis) [inline media-video/webcamoid/webcamoid-9.0.0.ebuild:16]: These too.

**Review by juippis — APPROVED** [review]
Thanks, I'll squash your commits and do some fixups, but good work here!

**Comment by joakim-tjernlund** [comment]
Maybe an an -9999 git src ebuild too?
Lots of fixes since 9.0.0 but webcamoid does not release often so would be great to have an -9999 too.

**Pull request closed without merging**